test/units/plugins/action/test_synchronize.py

```diff
@@ -99,11 +99,11 @@ class SynchronizeTester(object):
     execute_called = False
 
 
-    def _execute_module(self, module_name, task_vars=None):
+    def _execute_module(self, module_name, module_args=None, task_vars=None):
```

Comment:
Oh I see... this is a fake _execute_module() in the test case.  It's capturing information about how it was called.

So I think the way to adjust the test cases is to set ```self.final_module_args = module_args``` in this function and then adjust all of the assertions to use self.final_module_args instead of self.task._tmp_args.